flutter/shell/platform/tizen/channels/platform_channel.cc

```diff
@@ -158,10 +158,11 @@ void PlatformChannel::HandleMethodCall(
 }
 
 void PlatformChannel::SystemNavigatorPop() {
-  if (view_->GetType() == TizenViewType::kWindow) {
-    ui_app_exit();
+  auto* view = dynamic_cast<TizenView*>(view_);
+  if (view) {
```

Comment:
Note that this can be even simplified to
```suggestion
  if (auto* view = dynamic_cast<TizenView*>(view_)) {
```
if you prefer. (I don't know if it's a good practice in terms of code readability.)

Reply:
I updated code style.
Actually, I don't know if this is more good for readability. However, I think it might(?) be good because it makes it easier to know the scope of local variables.